webapp/src/Controller/API/AbstractRestController.php

```diff
@@ -137,11 +137,15 @@ protected function renderCreateData(
         $params = [
             'id' => $id,
         ];
+        $postfix = '';
         if ($routeType !== 'user') {
             $params['cid'] = $request->attributes->get('cid');
+            if ($params['cid'] === null) {
+                $postfix = '_1';
```

Comment:
what's happening in this file?

Reply:
This is broken currently already on main: If you create a team using `POST /api/teams` (and not `/api/contests/xxx/teams`, you will get a 500. This tries to fix this, but I can extract this in a separate commit if you want.